draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -167,20 +174,29 @@ const renderTag = (surveyStatus: SurveyStatus) => {
   )
 }
 
-const renderAvatar = (image: JSX.Element) => (
-  <div className={styles.avatar}>{image}</div>
+const renderAvatar = (image: JSX.Element, avatarAutomationId: string) => (
+  <div data-automation-id={avatarAutomationId} className={styles.avatar}>
+    {image}
+  </div>
 )
 
-const renderSubtitle = (subtitle: string) => (
+const renderSubtitle = (subtitle: string, subtitleAutomationId: string) => (
   <div className={styles.subtitle}>
-    <span className={styles.subtitleText}>{subtitle}</span>
+    <span
+      data-automation-id={subtitleAutomationId}
+      className={styles.subtitleText}
+    >
+      {subtitle}
+    </span>
   </div>
 )
 
 const renderSectionTitle = (
   sectionTitle?: string,
   sectionTitleDescription?: string,
-  variant?: Variant
+  variant?: Variant,
+  sectionTitleAutomationId?: string,
+  sectionTitleDescriptionAutomationId?: string
```

Comment:
Left as optional only as previous args were optional. It felt odd having them as the only required values (they will always be provided with the defaults).